plugins/jetty/src/main/java/com/navercorp/pinpoint/plugin/jetty/interceptor/ServerHandleInterceptor.java

```diff
@@ -0,0 +1,295 @@
+                }
+            } else {
+                if (isDebug) {
+                    logger.debug("TraceID exist. camSampled is false. skip trace. traceId:{}, requestUrl:{}, remoteAddr:{}", new Object[] { traceId, request.getRequestURI(), request.getRemoteAddr() });
+                }
+            }
+            return trace;
+		} else {
+			final Trace trace = traceContext.newTraceObject();
+            if (trace.canSampled()) {
+                SpanRecorder recorder = trace.getSpanRecorder();
+                recordRootSpan(recorder, request);
+                setTraceMetadata(request, trace);
+                if (isDebug) {
+                    logger.debug("TraceID not exist. start new trace. requestUrl:{}, remoteAddr:{}", request.getRequestURI(), request.getRemoteAddr());
+                }
+            } else {
+                if (isDebug) {
+                    logger.debug("TraceID not exist. camSampled is false. skip trace. requestUrl:{}, remoteAddr:{}", request.getRequestURI(), request.getRemoteAddr());
+                }
+            }
+            return trace;
+		}
+	}
+
+	@Override
+	public void after(Object target, Object[] args, Object result, Throwable throwable) {
+        if (isDebug) {
+            logger.afterInterceptor(target, args, result, throwable);
+        }
+
+		final Trace trace = traceContext.currentRawTraceObject();
+		if (trace == null) {
+			return;
+		}
+        // TODO STATDISABLE this logic was added to disable statistics tracing
+        if (!trace.canSampled()) {
+            traceContext.removeTraceObject();
+            return;
+        }
+		// ------------------------------------------------------
+		try {
+			SpanEventRecorder recorder = trace.currentSpanEventRecorder();
+			final HttpChannel<?> channel = (HttpChannel<?>) args[0];
+			final HttpServletRequest request = (HttpServletRequest) channel.getRequest();
+			final String parameters = getRequestParameter(request, 64, 512);
+			if (parameters != null && parameters.length() > 0) {
+				recorder.recordAttribute(AnnotationKey.HTTP_PARAM, parameters);
+			}
+
+			recorder.recordApi(methodDescriptor);
+			recorder.recordException(throwable);
+		} catch (Throwable th) {
+			if (logger.isWarnEnabled()) {
+				logger.warn("after. Caused:{}", th.getMessage(), th);
+			}
+		} finally {
+			traceContext.removeTraceObject();
+			deleteTrace(trace, target, args, result, throwable);
+		}
+	}
+	
+	private boolean samplingEnable(HttpServletRequest request) {
+		// optional value
+		final String samplingFlag = request.getHeader(Header.HTTP_SAMPLED.toString());
+		if (isDebug) {
+			logger.debug("SamplingFlag:{}", samplingFlag);
+		}
+		return SamplingFlagUtils.isSamplingFlag(samplingFlag);
+	}
+
+	private void setTraceMetadata(final HttpServletRequest request, final Trace trace) {
+		if(traceAccessor.isApplicable(request)) {
```

Comment:
METADATA_TRACE is not injected to any class. This condition will always evaluated to false.